crates/uv/tests/help.rs

```diff
@@ -230,8 +229,6 @@ fn help_subcommand() {
     success: true
     exit_code: 0
     ----- stdout -----
-
-    ----- stderr -----
```

Comment:
Interesting this changes the default display from stderr to stdout — does this matter to us? It should match Cargo's behavior and seems okay to me.

Reply:
Help output is regular is regular output that the user should read so that sounds correct to me

Reply:
Updated to use stdout consistently